<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="png/icons.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_WidgetPen</class>
  <widget class="QWidget" name="QG_WidgetPen">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>236</width>
        <height>121</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Pen</string>
    </property>
    <layout class="QGridLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>
      <item row="0" column="0">
        <widget class="Q3ButtonGroup" name="bgPen">
          <property name="title">
            <string>Pen</string>
          </property>
          <layout class="QGridLayout">
            <item row="3" column="1">
              <spacer name="spacer9">
                <property name="sizeHint">
                  <size>
                    <width>21</width>
                    <height>1</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="lLineType">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>Line type:</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <spacer name="spacer8">
                <property name="sizeHint">
                  <size>
                    <width>21</width>
                    <height>1</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="lWidth">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>Width:</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="QG_LineTypeBox" name="cbLineType"/>
            </item>
            <item row="0" column="1">
              <widget class="QG_ColorBox" name="cbColor"/>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="lColor">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>Color:</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="QG_WidthBox" name="cbWidth"/>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>QG_ColorBox</class>
      <extends>QWidget</extends>
      <header location="local">qg_colorbox.h</header>
      <sizehint>
        <width>140</width>
        <height>22</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>1</hordata>
        <verdata>0</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>QG_WidthBox</class>
      <extends>QWidget</extends>
      <header location="local">qg_widthbox.h</header>
      <sizehint>
        <width>140</width>
        <height>22</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>1</hordata>
        <verdata>0</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>QG_LineTypeBox</class>
      <extends>QWidget</extends>
      <header location="local">qg_linetypebox.h</header>
      <sizehint>
        <width>140</width>
        <height>22</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>1</hordata>
        <verdata>0</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0">
      <data format="XPM.GZ" length="45">789cd3d7528808f055d0d2e72a2e492cc94c5648ce482c52d04a29cdcdad8c8eb5ade6523250004143a55a6b2e0026630c4f</data>
    </image>
  </images>
  <includes>
    <include location="local">rs_pen.h</include>
    <include location="local">qg_linetypebox.h</include>
    <include location="local">qg_colorbox.h</include>
    <include location="local">qg_widthbox.h</include>
  </includes>
</ui>
